src/renderer_d3d11.cpp

```diff
@@ -1035,7 +1035,11 @@ namespace bgfx { namespace d3d11
 						: DXGI_SCALING_STRETCH
 						;
 					m_scd.swapEffect = m_swapEffect;
+#if BX_PLATFORM_WINRT
+					m_scd.alphaMode  = (_init.resolution.reset & BGFX_RESET_TRANSPARENT_BACKBUFFER) ? DXGI_ALPHA_MODE_PREMULTIPLIED : DXGI_ALPHA_MODE_IGNORE;
```

Comment:
Preferred form is (reduces ifdefery):

```
if (BX_ENABLED(BX_PLATFORM_WINRT) )
{
    m_scd.alphaMode  = (_init.resolution.reset & BGFX_RESET_TRANSPARENT_BACKBUFFER) ? DXGI_ALPHA_MODE_PREMULTIPLIED : DXGI_ALPHA_MODE_IGNORE;
}
else
{
    m_scd.alphaMode = DXGI_ALPHA_MODE_IGNORE;
}
```

Or even better:

```
m_scd.alphaMode  = (_init.resolution.reset & BGFX_RESET_TRANSPARENT_BACKBUFFER) ? DXGI_ALPHA_MODE_PREMULTIPLIED : DXGI_ALPHA_MODE_IGNORE;
```

And then on `bgfx::reset` side you can validate that when user specify `BGFX_RESET_TRANSPARENT_BACKBUFFER`, `BGFX_CAPS_TRANSPARENT_BACKBUFFER` is available.